tasks/main.yml

```diff
@@ -18,10 +18,11 @@
 - name: Install Docker's dependencies
   apt:
     name: "{{ docker__package_dependencies + docker__pip_dependencies }}"
+    force_apt_get: true
 
 - name: Add Docker's public PGP key to the APT keyring
   apt_key:
-    id: "{{ docker__apt_key_id }}"
+    #    id: "{{ docker__apt_key_id }}"
```

Comment:
What was your reasoning behind removing this?

Reply:
id in this case is excessive, id need for apt key server